packages/places/android/src/main/java/com/adobe/marketing/mobile/reactnative/places/RCTAEPPlacesModule.java

```diff
@@ -43,7 +43,7 @@ public RCTAEPPlacesModule(ReactApplicationContext reactContext) {
 
   @Override
   public String getName() {
-    return "RCTAEPPlaces";
+    return "AEPPlaces";
```

Comment:
Good catch! @namArora3112

Reply:
raising a separate PR for the same . isolating tsx and places fixes with different PR